src/azle.ts

```diff
@@ -34,23 +34,42 @@ const GLOBAL_AZLE_TARGET_DIR = `${GLOBAL_AZLE_CONFIG_DIR}/target`;
 azle();
 
 function azle() {
+    const install_rustup_path = path.resolve(__dirname, '../install_rustup.sh');
+
+    // TODO research exec vs spawn
+    execSync(`${install_rustup_path} ${version} ${rust_version}`, {
+        stdio: 'inherit'
+    });
```

Comment:
Running this script could fail. We should add error handling and tell them to check the logs directory.